src/components/base/Tag/Tag.module.scss

```diff
@@ -0,0 +1,26 @@
+@use "../../../styles/mixins/mixins-typo" as *;
+@use "../../../styles/utilities/functions" as *;
+@use "../../../styles/utilities/utilities" as *;
+
+.tag {
+  @include label-text;
+
+  --spacing-block: 6;
+
+  display: inline-block;
+  padding: px-to-rem(var(--spacing-2xs)) px-to-rem(var(--spacing-block));
+
+  @include forced-colors {
+    border: px-to-rem(var(--border-width-s)) solid LinkText;
```

Comment:
It does not matter much at this point but I would probaly go with CanvasText here.